### PR TITLE
Tests: Added tests for .NET Core 1.0 and 2.0 apps

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs [hunk 1]
```diff
@@ -37,6 +37,49 @@ public DotNetCoreEndToEndTests(ITestOutputHelper output, TestTempDirTestFixture
...
+                NetCoreApp11WebApp,
+                _output,
+                volume,
+                "/bin/sh",
+                new[]
+                {
+                    "-c",
+                    buildImageScript
+                },
+                "oryxdevms/dotnetcore-1.0",
```
Comment (kichalla) [inline tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs]: The runtime image is still 1.0

File: tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs [hunk 2]
```diff
@@ -124,6 +167,49 @@ public async Task CanBuildAndRun_NetCore11WebApp_HavingExplicitAssemblyName()
...
+        public async Task CanBuildAndRun_NetCoreApp20WebApp()
+        {
+            // Arrange
+            var hostDir = Path.Combine(_hostSamplesDir, "DotNetCore", "aspnetcore20");
+            var volume = DockerVolume.Create(hostDir);
+            var appDir = volume.ContainerDir;
+            var appOutputDir = $"{appDir}/myoutputdir";
+            var buildImageScript = new ShellScriptBuilder()
+                // NOTE: we are using 2.1 'sdk' version to build 2.0 netcoreapp
+                .AddCommand($"oryx build {appDir} -l dotnet --language-version 2.1 -o {appOutputDir}")
```
Comment (kichalla) [inline tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs]: Similarly this is a 2.0 runtime app built with 2.1 SDK

File: tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs [hunk 3]
```diff
@@ -124,6 +167,49 @@ public async Task CanBuildAndRun_NetCore11WebApp_HavingExplicitAssemblyName()
...
+                NetCoreApp11WebApp,
+                _output,
+                volume,
+                "/bin/sh",
+                new[]
+                {
+                    "-c",
+                    buildImageScript
+                },
+                "oryxdevms/dotnetcore-2.0",
```
Comment (kichalla) [inline tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs]: The runtime image is still 2.0

File: tests/SampleApps/DotNetCore/aspnetcore10/Controllers/HomeController.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace app.Controllers
```
Comment (kichalla) [inline tests/SampleApps/DotNetCore/aspnetcore10/Controllers/HomeController.cs]: This apps were taken from the share that @JennyLawrance shared with us

**Comment by dorfire** [comment]
Should we have a `SOURCE.md`?

**Comment by kichalla** [comment]
> Should we have a SOURCE.md?

Good question. These apps were from Jenny, so we do not know their source. From their structure it looks like they are dotnet core cli template-based projects, so we are good.

**Pull request merged**